cluster-operator/src/main/java/io/strimzi/operator/cluster/model/StorageDiff.java

```diff
@@ -133,19 +129,10 @@ private StorageDiff(Reconciliation reconciliation, Storage current, Storage desi
                     }
                 }
 
-                // Some changes to overrides are allowed:
-                // * When scaling up or down, you can set the overrides for new nodes
-                // * You can set overrides for nodes which do nto exist (yet)
-                if (pathValue.startsWith("/overrides")) {
-                    if (isOverrideChangeAllowed(current, desired, currentNodeIds, desiredNodeIds))    {
-                        continue;
-                    }
-                }
-
-                if (LOGGER.isDebugEnabled()) {
-                    LOGGER.debugCr(reconciliation, "Storage {}differs: {}", volumeDesc, d);
-                    LOGGER.debugCr(reconciliation, "Current Storage {}path {} has value {}", volumeDesc, pathValue, lookupPath(source, pathValue));
-                    LOGGER.debugCr(reconciliation, "Desired Storage {}path {} has value {}", volumeDesc, pathValue, lookupPath(target, pathValue));
+                if (LOGGER.isInfoEnabled()) {
+                    LOGGER.infoCr(reconciliation, "Storage {}differs: {}", volumeDesc, d);
+                    LOGGER.infoCr(reconciliation, "Current Storage {}path {} has value {}", volumeDesc, pathValue, lookupPath(source, pathValue));
+                    LOGGER.infoCr(reconciliation, "Desired Storage {}path {} has value {}", volumeDesc, pathValue, lookupPath(target, pathValue));
```

Comment:
why are we logging at info level now? INFO is the default I guess, so it will be always logged. Do we really need that?